desktop/appcontainer/ExceptionDialog.js

```diff
@@ -27,7 +29,12 @@ export const exceptionDialog = hoistCmp.factory({
     render({model}) {
         const {exception, options} = model;
 
-        if (!exception) return null;
+        if (!exception) {
+            FocusStyleManager.onlyShowFocusOnTabs();
+            return null;
+        }
+
+        FocusStyleManager.alwaysShowFocus();
```

Comment:
This `FocusStyleManager. alwaysShowFocus();` is necessary only when programmatically setting focus on an element.   It wasn't necessary when autoFocus:true.